sampler/src/AgateEnvironment/AgateEnvironment.js

```diff
@@ -129,22 +143,34 @@ const StorybookDecorator = (story, config) => {
 		}
 	};
 	const skinFromURL = getPropFromURL('skin');
-	const currentSkin = skinFromURL ? skinFromURL : 'gallium';
+	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
 	const newSkin = (memory.skin !== currentSkin);
 	memory.skin = currentSkin;  // Remember the skin for the next time we load.
 	const accentFromURL = getPropFromURL('accent');
 	const highlightFromURL = getPropFromURL('highlight');
+	const allSkins = boolean('show all skins', Config);
+	const skinKnobs = {};
+	if (!allSkins) {
+		skinKnobs.skin = select('skin', skins, Config, currentSkin);
+	}
 
 	return (
 		<Agate
 			title={`${config.kind} ${config.story}`.trim()}
 			description={config.description}
-			skin={select('skin', skins, Config, currentSkin)}
-			skinVariants={boolean('Night Mode', Config, false) && 'night'}
+			{...skinKnobs}
+			skinVariants={boolean('night mode', Config) && 'night'}
```

Comment:
Are `skinVariants` not rendered out as part of the skin?  Do all skins support variants?  Does it matter?  Also, space in knob name.

Reply:
1. Variants are indeed rendered as part of the skin
2. Not all skins support variants
3. Skins are _allowed_ to not support a particular variant
4. knobs with spaces typically relate to things that don't 1:1 correlate with real props. If this were a real prop, it would be called `skinVariants` and have a different type of knob.